### DISCUSSION TITLE
Pasting from Evernote removes multi line formatting

### DISCUSSION THREAD

**Discussion by goldblatt** [facebook/lexical#4304] [Q&A]
Hey all--

I have a use case for copying notes over from evernote. However whenever I copy something over that includes multiple lines-- the lexical editor puts it all on one line. I've tested on the lexical playground + my dev and neither work. I've also noticed that if i paste into google docs/gmail/github editor/etc it works fine.

Eg
```
writing a  note
a second thing
woooohooo
```
becomes 
`writing a  notea second thingwoooohooo 
`
When I investigate a little more I can see that that evernote editor uses divs to implement paragraphs and the meta/html data in my paste is the following:
`<meta charset='utf-8'><div data-pm-slice="1 1 []" data-en-clipboard="true">writing a  note</div><div>a second thing</div><div>woooohooo</div>`

I don't really want to rewrite the code the handles pasting and was hoping there might be a simple fix for this. I have the following to help debug but I'm not totally sure how to go forward here without rewriting a lot of the paste functionality which seems dicey. Wondering if anyone else has run into this issue or has ideas
```
editor.registerCommand<
    InputEvent & ClipboardEvent
  >(
    PASTE_COMMAND,
    (event) => {
      for (var i = 0; i < event.clipboardData.items.length; i++) {
        event.clipboardData?.items[i].getAsString((s) =>
          console.log(s)
        );
      }
      return false;
    },
    COMMAND_PRIORITY_EDITOR
  );
```

**Comment by moy2010** [comment]
I have seen this when copying from Discord as well. You might want to take a look into the `importDOM` method in `LexicalTextNode.ts` and try to handle the `div` case.

**Comment by goldblatt** [comment]
okay so I was able to fix this, albeit not how I imagine it should be fixed but the lexical editor has an _htmlConversions internal var that handles this. If i take that and add the following it'll treat div's like p. This is kind of weird so not sure if it's really the dream solution. I think you can pass this into the config. I will also note that I'm having a more concerning issue with paste going out of order when posting from notion. But the code below should fix the div/paragraph issue

```
editor._htmlConversions.set("div", editor._htmlConversions.get("p"));
```
